setup.py

```diff
@@ -13,7 +13,7 @@ def read(fname):
     install_requires = ['requests>=2.1.0', 'streql>=3.0.2'],
 
     url = 'https://github.com/bigcommerce/bigcommerce-api-python',
-    download_url = 'https://github.com/bigcommerce/bigcommerce-api-python/releases',
+    download_url = 'https://pypi.python.org/packages/source/b/bigcommerce/bigcommerce-0.13.0.tar.gz',
```

Comment:
This will require you to update the `download_url` as well as the `version` when you cut a new release. What about moving the version number out of the kwargs of setup?:

    VERSION = '0.13.0'

    setup(
        version = VERSION,
        download_url = 'https://pypi.python.org/packages/source/b/bigcommerce/bigcommerce-{}.tar.gz'.format(version),
        ...
        )